src/main/java/com/hugudungs/hugupjigup/data/entity/board/FreeText.java

```diff
@@ -0,0 +1,33 @@
+package com.hugudungs.hugupjigup.data.entity.board;
+
+import com.hugudungs.hugupjigup.data.entity.user.User;
+import jakarta.persistence.*;
+import lombok.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "free_text")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@AttributeOverrides({
+        @AttributeOverride(name = "id", column = @Column(name = "free_text_id", nullable = false)),
+        @AttributeOverride(name = "title", column = @Column(name = "free_title", nullable = false)),
+        @AttributeOverride(name = "content", column = @Column(name = "free_text", nullable = false, columnDefinition = "TEXT")),
+        @AttributeOverride(name = "views", column = @Column(name = "free_number", nullable = false, columnDefinition = "INT DEFAULT 0"))
```

Comment:
```suggestion
        @AttributeOverride(name = "id", column = @Column(name = "free_id", nullable = false)),
        @AttributeOverride(name = "title", column = @Column(name = "free_title", nullable = false)),
        @AttributeOverride(name = "content", column = @Column(name = "free_content", nullable = false, columnDefinition = "TEXT")),
        @AttributeOverride(name = "views", column = @Column(name = "free_views", nullable = false, columnDefinition = "INT DEFAULT 0"))
```
위와 같은 이유로 컬럼 이름 변경해줬으면 하고 어지간하면 필드이름과 컬럼 이름은 통일성 있게 가는게 헷갈리지 않을거야